vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/ItemClickListenerIT.java

```diff
@@ -63,22 +63,34 @@ public void clickCheckboxInCell_noItemClickEventFired() {
         Assert.assertEquals("", getClickMessage());
     }
 
-    @Test
-    public void clickCell_clickCheckboxInCell_onlyOneClickEventFired() {
```

Comment:
Why did you remove an existing test?

Reply:
not my day :| will bring it back with the next commit... *sigh* good there are code reviews